demo/agent.cc

```diff
@@ -29,6 +31,11 @@ bool ParseCommandLine(int argc, char* argv[]) {
     if (arg.find(kArgUserSpecific) == 0) {
       path = kPathUser;
       user_specific = true;
+    } else if (arg.find(kArgDelaySpecific) == 0) {
```

Comment:
To double check it is intentional: if you assign values to `path` & `user_specific`, you can't assign a value to `delay`?

Reply:
I just tested and it is possible to set both.  This if / else if / else block is inside a fop loop that goes over al the args, so it should handle all of them.  Yes/no?

Reply:
Makes sense, thanks for the explanation!